src/components/elements/typography/index.js

```diff
@@ -0,0 +1,115 @@
+import { Link as GatsbyLink } from 'gatsby'
+import styled from 'styled-components'
+import { letterSpacing, colors, size, psize, lineHeight } from '../../../utils'
+
+const { blueCharcoal, mineShaft } = colors
+
+export const H1 = styled.h1`
+  font-size: ${size.xl};
+  letter-spacing: ${letterSpacing.medium};
+  white-space: nowrap;
+  font-weight: 800;
+`
+
+export const H2 = styled.h2`
+  font-size: ${size.lg};
+  letter-spacing: ${letterSpacing.medium};
+  font-weight: 700;
+`
+
+export const H3 = styled.h3`
+  font-size: ${size.md};
+  letter-spacing: ${letterSpacing.small};
+  font-weight: 700;
+`
+
+export const H4 = styled.h4`
+  font-size: ${size.xs};
+  letter-spacing: ${letterSpacing.small};
+  font-weight: 700;
+`
+
+export const CapsHeadingLarge = styled.h1`
+  font-size: ${size.s};
+  letter-spacing: ${letterSpacing.huge};
+  white-space: nowrap;
```

Comment:
I don't think this is needed as there aren't any vertical cap headings, same with the other ones.